plsql/ExcelGen.pkb

```diff
@@ -72,6 +72,7 @@ create or replace package body ExcelGen is
   ST_STRING              constant pls_integer := 1;
   ST_DATETIME            constant pls_integer := 2;
   ST_LOB                 constant pls_integer := 3;
+  ST_BINARY_DOUBLE       constant pls_integer := 4;
```

Comment:
It's not necessary to define another supertype for BINARY_DOUBLE. It's actually handled like a NUMBER.